src/OAuth2/ResponseType/JwtAccessToken.php

```diff
@@ -127,5 +127,18 @@ protected function createPayload($client_id, $user_id, $scope = null)
             'token_type' => $this->config['token_type'],
             'scope'      => $scope
         );
+        
+        if (isset($this->config['jwt_extra_payload'])) {
+            if (is_callable($this->config['jwt_extra_payload'])) {
+                $extra = call_user_func($this->config['jwt_extra_payload'], $client_id, $user_id, $scope);
+                if (is_array($extra)) {
+                    $payload = array_merge($extra, $payload);
+                }
+            } else {
+                throw new \InvalidArgumentException("config['jwt_extra_payload'] is not callable");
```

Comment:
because of `if (isset($this->config['jwt_extra_payload'])) ` will throw exception on empty values like `null, '', 0 or false`